### PR TITLE
Add farewell files

### PR DESCRIPTION
**Pull request by mcferrod19**

### CODE REVIEW

**Review by guerrero — APPROVED** [review]
👍

File: src/pages/farewell/farewell.sccs [hunk 1]
```diff
@@ -0,0 +1,4 @@
+page-home {
```
Comment (gootyfer) [inline src/pages/farewell/farewell.sccs:1]: It should be `farewell-home`, right?

File: src/pages/farewell/farewell.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import { Component } from '@angular/core';
+import { NavController } from 'ionic-angular';
+
+@Component({
+  selector: 'page-farewell',
+  templateUrl: 'farewell.html'
+
+  
+})
+export class HomePage {
```
Comment (gootyfer) [inline src/pages/farewell/farewell.ts:10]: The name of this class should be `FarewellPage`

**Review by gootyfer — COMMENTED** [review]
Just a couple of naming errors. Good work creating a new page!

File: src/pages/farewell/farewell.sccs (hunk 1 above)
Comment (mcvcamblor) [reply src/pages/farewell/farewell.sccs:1]: No. Farewell is the last page and home is the first

Comment (gootyfer) [reply src/pages/farewell/farewell.sccs:1]: Got it! But this is the `.scss` file for the farewell (last page), isn't it?

Comment (mcvcamblor) [reply src/pages/farewell/farewell.sccs:1]: Yes. It is the las one page but we won't write in this file

**Pull request merged**